src/breadcrumb/breadcrumb.style.tsx

```diff
@@ -106,14 +106,16 @@ export const Item = styled.li<ItemStyleProps>`
 `;
 
 export const Caret = styled(ChevronRightIcon)`
-    margin: 0.25rem 0.25rem 0.1rem 0.25rem;
-    color: ${V2_Color.Neutral[4]};
+    margin: 0.25rem;
+    height: 1em;
```

Comment:
for this to work the parent needs to set font size. can apply the `body-size-md` token to the `ul` parent